Sources/Apollo/MultipartFormData.swift

```diff
@@ -0,0 +1,111 @@
+public class MultipartFormData {
```

Comment:
@designatednerd will it make sense to make this internal? 
As it is used for internal stuff?

Reply:
You'll need to `@testable import Apollo` in the tests in order to access it, but you're right, this is an implementation detail. Great catch!

Reply:
Will remove `public`. 🚀